docs/TuxActionStores.md

```diff
@@ -0,0 +1,144 @@
+#Tux ActionStores
+##Table of Contents
+<ol>
+  <li><a href="#Context">Context</a></li>
+  <li><a href="#Premise">Premise</a></li>
+  <li><a href="#Implementation">Implementation</a></li>
+    <ol style="list-style-type:upper-alpha">
+      <li><a href="#Requiring-ActionStores">Requiring ActionStores</a></li>
+      <li><a href="#Creating-Action-Stores">Creating ActionStores</a>
+      </li>
+      <li><a href="#TuxActionStores-Complete-Example">Tux ActionStores Complete Example</a></li>
+    </ol>
+</ol>
+
+## <a id="Context"></a>Context [#](#Context)
+Recall that [Tux Actions](https://tuxedojs.org/docs/TuxActions) allow you to create, listen, and dispatch actions as per the Tux architecture.
+
+```
+    var Actions = require('tux/Actions');
+
+    var messageActions = Actions.createActionCategory({
+      category: 'messages',
+      source: 'message views',
+      actions: ['get', 'create', 'delete']
+    });
+```
+
+ Well, with ActionStores, we provide an alternative approach to registering your stores with your actions.
+
+## <a id="Premise"></a>Premise [#](#Premise)
+>Tux `ActionStores` is a module providing the interface for creating stores that are pre-packaged with a 'register' convenience method for automatically registering the store with actions.
+
+## <a id="Implementation"></a>Implementation [#](#Implementation)
+### <a id="Requiring-ActionStores"></a>Requiring ActionStores [#](#Requiring-Action-Stores)
+
+The interface for `ActionStores` is exposed via a constructor function:
```

Comment:
Change to ...via a `createStore` constructor function on the `ActionStores` object.